Formula/jenkinsfile-runner.rb

```diff
@@ -0,0 +1,15 @@
+class JenkinsfileRunner < Formula
+    desc "A command line tool to run Jenkinsfile"
```

Comment:
Change it to "Jenkinsfiles" plural I think - reads a bit nice.

Reply:
Nice catch. I agree.